helix-query.yaml

```diff
@@ -41,3 +41,28 @@ indices:
       cq-tags:
         select: head > meta[name="cq-tags"]
         value: attribute(el, "content")
+  search:
+    include:
+      - /**
+    exclude:
+      - /configuration
+      - /nav
+      - /footer
+      - /fragments
+    target: /query-index.json
+    properties:
```

Comment:
@mayur-satav, could you also include the 'cq:tags' property for better search results accuracy?

Reply:
Sure

Reply:
Add following as well to block non index pages from search:

`robots:
        select: head > meta[name="robots"]
        value: attribute(el, "content")`